modules/misunderstood/src/views/full/index.tsx

```diff
@@ -354,6 +403,23 @@ export default class MisunderstoodMainView extends React.Component<Props, State>
               {lang.tr('module.misunderstood.qnaThumbsDown').toUpperCase()}
             </Button>
           </div>
+          <div className={style.filterContainer}>
+            <Button onClick={this.clickImportData}>{lang.tr('module.misunderstood.import')}</Button>
+            <Button
+              disabled={this.state.eventCounts ? !this.state.eventCounts[FLAGGED_MESSAGE_STATUS.applied] : true}
+              onClick={this.exportData}
+            >
+              {lang.tr('module.misunderstood.export')}
+            </Button>
+            <input
+              style={{ display: 'none' }}
+              ref={this.importInput}
+              type={'file'}
+              disabled={false}
+              onChange={ev => this.importData(ev.target)}
+              accept={'application/json'}
```

Comment:
1. Please use FileInput form @blueprintjs/core instead, so we have a similar look & feel
2. In tsx files, prefer double quotes instead (eg: `type="file"` instead of `type={'file'}` 
3. disabled={false} is superfluous and can be removed